autoscaler/cluster.py

```diff
@@ -536,7 +536,11 @@ def fulfill_pending(self, asgs, selectors_hash, pods):
 
             new_capacity = group.actual_capacity + units_requested
             if not self.dry_run:
-                async_operation = group.scale(new_capacity)
+                try:
+                    async_operation = group.scale(new_capacity)
+                except CloudError as e:
+                    logger.warn("Error while scaling Scale Set: {}".format(e.message))
```

Comment:
ya, I believe the sentry credentials are already configured. we removed the client at one point because it wasn't being used, but should be straight-forward to plug back in.